code-studio/src/js/components/AdvancedShareOptions.jsx

```diff
@@ -62,6 +68,26 @@ var AdvancedShareOptions = React.createClass({
     );
   },
 
+  renderEmbedTab() {
+    var url = window.location.href.replace('edit','embed');
+    var iframeHtml = '<iframe width="352" height="612" style="border: 1px solid black;" src="' +
+          url + '"></iframe>';
+    return (
+      <div>
+        <p style={style.p}>
+          You can paste the embed code into an HTML page to display
+          the project on a webpage.
```

Comment:
Actually, I can't find any docs on this.  This commit is a good example though: https://github.com/code-dot-org/code-dot-org/commit/16b5e77978b0cd8b076d323c5ae90f002aed73ec